docs/python/jupyter-support.md

```diff
@@ -125,13 +125,14 @@ The Visual Studio Code debugger lets you step through your code, set breakpoints
 
 1. Import the notebook's `.ipynb` file into VS Code as described in the previous section. (Download the file first if you're using a cloud-based Jupyter environment such as [Azure Notebooks](https://docs.microsoft.com/azure/notebooks).)
 
-1. Follow the instructions to configure and run the debugger as described on [Tutorial - Configure and run the debugger](/docs/python/python-tutorial.md#configure-and-run-the-debugger), using your imported `.ipynb` file, of course, and setting a breakpoint in an appropriate location in your notebook code.
+1. To start the debugger, use one of the following options:
 
-1. For an individual cell, you can also start the debugger using the **Debug Cell** adornment that appears above the cell. Using the adornment starts the debugger specifically on the code in that cell.
+    - For the whole notebook, open the Command Palette (`kbstyle(Ctrl+Enter)`) and run the **Python: Debug Current File in Python Interactive Window** command.
```

Comment:
@kraigb This doesn't look right. Shouldn't this be `kb(workbench.action.showCommands)`?

Reply:
Oh, you're right...I copied the wrong snippet. Let me fix that.

Reply:
@gregvanl Check https://github.com/microsoft/vscode-docs/pull/2982 for me, and merge it after you do.